src/mgl-pax.el

```diff
@@ -177,13 +179,14 @@ other mgl-pax commands in interactive use."
 ;;; If the ASDF-SYSTEM is `:mgl-pax/document' and `(mgl-pax-use-w3m)`
 ;;; is nil, then `:mgl-pax/web' is loaded instead and the web server
 ;;; is started.
-(cl-defmacro mgl-pax-with-component ((asdf-system) &body body)
+(cl-defmacro mgl-pax-with-component
+    ((asdf-system &optional (interactivep `(called-interactively-p 'any))) &body body)
```

Comment:
Ah, I see. This makes the fragile logic with `interactivep` more visible and overrideable. Thanks.